.github/workflows/full-check.yml

```diff
@@ -29,7 +29,8 @@ jobs:
       - name: Checkout GWT tools into a sibling directory
         uses: actions/checkout@v4
         with:
-          repository: 'gwtproject/tools'
+          repository: 'vegegoku/tools'
+          ref: 'no-repackaging-jdt-test'
```

Comment:
I think we just need to revert these temp workarounds and this should be ready for merge?